Products/PloneKeywordManager/browser/prefs_keywords_view.pt

```diff
@@ -177,11 +177,11 @@
                                  onclick string:document.forms['keyword_edit_form'].changeto.value='${subject}';; return true;;" />
           <label tal:content="subject" tal:attributes="for string:subject-${subject_id};">Subject</label>
           <tal:p5 tal:condition="view/is_plone_5">
-            <a href="#" tal:attributes="href string:${portal_url}/@@search?Subject=${subject}">
+            <a href="#" tal:attributes="href string:${portal_url}/@@search?${field}=${subject}" target="_blank">
```

Comment:
in general, opening a new window is a basic usability mistake that must be avoided: see item 9 of: https://www.nngroup.com/articles/top-10-mistakes-web-design/

opening a modal would be better.